android/app/src/main/java/com/mattermost/rnbeta/CustomPushNotification.java

```diff
@@ -173,19 +179,40 @@ protected void postNotification(int id, Notification notification) {
             style.setBigContentTitle(title);
             notification.setStyle(style)
                     .setContentTitle(summaryTitle);
-//                    .setNumber(numMessages);
         }
 
         // Let's add a delete intent when the notification is dismissed
-        Intent delIntent = new Intent(mContext, NotificationDismissReceiver.class);
+        Intent delIntent = new Intent(mContext, NotificationDismissService.class);
         delIntent.putExtra(NOTIFICATION_ID, notificationId);
-        PendingIntent deleteIntent = PendingIntent.getBroadcast(mContext, 0, delIntent, 0);
+        PendingIntent deleteIntent = NotificationIntentAdapter.createPendingNotificationIntent(mContext, delIntent, mNotificationProps);
         notification.setDeleteIntent(deleteIntent);
 
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
             notification.setGroup(GROUP_KEY_MESSAGES);
         }
 
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
```

Comment:
Wondering if I should add in the condition that postId es not null or do I leave it like this, leaving it as it is will let the user post to the same channel but not as a reply.

Thoughts?